src/operators.jl

```diff
@@ -264,6 +264,19 @@ end
 Base.isapprox(p::_APL, α; kwargs...) = isapprox(promote(p, α)...; kwargs...)
 Base.isapprox(α, p::_APL; kwargs...) = isapprox(promote(p, α)...; kwargs...)
 
+MA.operate!(::typeof(-), ::AbstractTermLike) = error("not implemented yet")
+
+MA.operate_to!(::AbstractTermLike, ::typeof(-), ::_APL) = error("not implemented yet")
+
+function MA.operate!(::typeof(-), p::_APL)
+    negate!! = x -> MA.operate!!(-, x)
+    return map_coefficients!(negate!!, p, nonzero = true)
```

Comment:
You can use `Base.Fix1(MA.operate!!, -)` to avoid creating a closure that might allocate

Reply:
> You can use `Base.Fix1(MA.operate!!, -)`

I prefer to never use either `Fix1` or `Fix2`, because I can never remember which is which. Should I?

> to avoid creating a closure that might allocate

In this case this is not an issue, as the closure doesn't capture any variables.